plugins/common/tls/client.conf

```diff
@@ -7,7 +7,16 @@
   # tls_cert = "/path/to/certfile"
   ## Used for TLS client certificate authentication
   # tls_key = "/path/to/keyfile"
+  ## Password for the key file if it is encrypted
+  # tls_key_pwd = ""
   ## Send the specified TLS server name via SNI
   # tls_server_name = "kubernetes.example.com"
+  ## Minimal TLS version to accept by the client
+  # tls_min_version = "TLS12"
+  ## List of ciphers to accept, by default all secure
+  ## ciphers will be accepted
+  # tls_cipher_suites = []
```

Comment:
Do we have a way to point at what is "secure" or the default list?

Reply:
The definition is [here](https://github.com/golang/go/blob/16ce8b3925deaeb72541ee96b6ee23a08fc21dea/src/crypto/tls/cipher_suites.go#L53) but there is no location we can reference. I'm a bit hesitant to put a list into the `sample.conf` as this might change between Golang versions. Should I add an output for the case where an invalid cipher is used?